services/config/config.go

```diff
@@ -49,6 +50,7 @@ type Config struct {
 	Services struct {
 		Platform  platform.Config         `yaml:"platform"`
 		Telemetry telemetry.ServiceConfig `yaml:"telemetry"`
+		Pprof     pprof.Config            `yaml:"pprof"`
```

Comment:
please remove it from the config, it will simplify the code a lot.

Reply:
done